### DISCUSSION TITLE
Switching windows light/dark theme automatically?

### DISCUSSION THREAD

**Discussion by V17** [Tyrrrz/LightBulb#260] [General]
First of all, thank you for this great application. I dislike how opinionated f.lux is regarding setting up some existing features or refusing to implement others, so LightBulb seems to be what I was looking for.

That said, there is a f.lux feature that I'm missing: switching windows from light to dark theme and back together with changing the color temperature and brightness. I can use AutoDarkMode, it seems to be working (better than f.lux even, which did not switch the task bar for some reason), so this is not some killer feature that I need to be happy. However since it's a task that one usually wants to do around the same time with the darkening, it would be really nice to set it up in one place and not have to install a separate app with unnecessary features.

So, just saying that if you ever considered implementing this feature, I'm giving it a small and insignificant +1.

**Comment by Tyrrrz** [comment]
Thank you. I don't have plans to significantly expand the feature set of LightBulb for now, so using AutoDarkMode is probably your best bet. Thank you for the suggestion regardless.
